docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+type: docs
+title: "Azure Automanage"
+linkTitle: "Azure Automanage"
+weight: 9
+description: >
+---
+
+## Enable Azure Automanage on an Azure Arc enabled server using an ARM template
+
+The following README will guide you on how to onboard an Azure Arc enabled server on to [Azure Automanage](https://docs.microsoft.com/en-us/azure/automanage/automanage-virtual-machines#prerequisites), so you can follow best practices in reliability, security and management for virtual machines using Azure services such as [Azure Update Management](https://docs.microsoft.com/en-us/azure/automation/update-management/overview) and [Azure Backup](https://docs.microsoft.com/en-us/azure/backup/backup-overview).
+
+Azure Automanage removes the need to discover virtual machines manually and automatically onboards and configures certain services in Azure following best practices as defined in [Microsoft Cloud Adoption Framework for Azure](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/manage/best-practices). Azure services included in Auzure Automation are: [Azure Backup](https://docs.microsoft.com/en-us/azure/backup/backup-overview), [Azure Security Center](https://docs.microsoft.com/en-us/azure/security-center/security-center-introduction), [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview) and [Azure Automation](https://docs.microsoft.com/en-us/azure/automation/automation-intro).
+
+In this guide, you will use an ARM template to enable and configure Azure Automation on your Azure Arc enabled servers.
```

Comment:
I think it needs to have a more descriptive explanation. Something like "By the end of this guide..." which also incl. Automanage.

Reply:
changed